README.rst

```diff
@@ -17,11 +17,58 @@ API
 Run locally
 ===========
 
+By default, readinglist persists its records inside a `Redis
+<http://redis.io/>`_  database, so you need to have it installed (see the
+"Install Redis" section below for more on this.
+
+Once you have Redis installed, you just need to do:
+
 ::
 
     make serve
 
 
+You can also change the configuration to persist everything in memory (not
+recommended). To do that, edit your `conf/readinglist.ini` file to have the
+following config::
+
+    readinglist.backend = readinglist.backend.memory
+
+
+
+Install Redis
+=============
+
+Linux
+-----
+
+On debian / ubuntu based systems:::
+
+    sh
```

Comment:
What is this `sh` thing ? I would rather put a $ in front of the following line or use 

    .. code-block:: sh